monai/apps/auto3dseg/data_analyzer.py

```diff
@@ -17,19 +17,23 @@
 
 import numpy as np
 import torch
+from torch.multiprocessing import Process, set_start_method, Manager
 
 from monai.apps.auto3dseg.transforms import EnsureSameShaped
 from monai.apps.utils import get_logger
 from monai.auto3dseg import SegSummarizer
 from monai.auto3dseg.utils import datafold_read
 from monai.bundle import config_parser
 from monai.bundle.config_parser import ConfigParser
-from monai.data import DataLoader, Dataset
+from monai.data import DataLoader, Dataset, partition_dataset
 from monai.data.utils import no_collation
 from monai.transforms import Compose, EnsureTyped, LoadImaged, Orientationd
 from monai.utils import StrEnum, min_version, optional_import
 from monai.utils.enums import DataStatsKeys, ImageStatsKeys
 
+import warnings
+# remove the warning "warnings.warn(f"Modifying image pixdim from {pixdim} to {norm}")"
+warnings.filterwarnings("ignore", category=UserWarning, module='monai')
```

Comment:
this will disable all user warnings generated by monai when the user run `import monai`... I'll change this to remove that particular warning from the util function